lstmcpipe/plot_irfs.py

```diff
@@ -0,0 +1,408 @@
+from astropy.table import QTable
+import matplotlib.pyplot as plt
+import numpy as np
+import astropy.units as u
+from pyirf.utils import cone_solid_angle
```

Comment:
Haven't work with pyirf a lot. Any incompatibilities between lstchain and pyirf ? should we specify the version for the installing

Reply:
Also @vuillaut, should we move the script to the `plot` package of the module ?

Reply:
Lstchain also uses pyirf and there shouldn't be a proble